packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/__tests__/useCopyView.spec.ts

```diff
@@ -21,9 +22,9 @@ describe('useCopyView', () => {
             prefix: 'test-prefix/',
             bucket: 'bucket',
             id: 'id',
-            permission: 'READWRITE',
+            permissions: ['delete', 'get', 'list', 'write'],
             type: 'PREFIX',
-          },
+          } as LocationData,
```

Comment:
nit: could we assign this to const with right permissions and use that variable here so we dont have to cast it ?

Reply:
TypeScript sometimes is dumb to think the permissions (e.g. `['list', 'write']`) to be readonly tuple instead of mutable (e.g. `('list'|'write')[]`). I can see if I can use const in other places.